controller/comments-controller.js

```diff
@@ -0,0 +1,14 @@
+const fetchComments = require("../model/comments-model");
+
+function getCommentsByArticleId(req, res, next) {
+  const { article_id } = req.params;
+  fetchComments(article_id)
+    .then((comments) => {
+      res.status(200).send(comments);
```

Comment:
It is best practice to send this in an object on a key of comments like {comments}